tests/tools/importexport/migration/test_v08_to_v09.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+"""Test export file migration from export version 0.8 to 0.9"""
+from aiida.tools.importexport.migration.v08_to_v09 import migrate_v8_to_v9
+
+from . import ArchiveMigrationTest
+
+
+class TestMigrate(ArchiveMigrationTest):
+    """Tests specific for this archive migration."""
+
+    def test_migrate_external(self):
```

Comment:
It may be useful to add a specific test for the `migration_dbgroup_type_string` function, to assert that you get the expected 1:1 conversion here. However, it may also be overkill, I'm not entirely sure :)

Reply:
Done